exporter.go

```diff
@@ -189,6 +189,14 @@ func escapeMetricName(metricName string) string {
 	return metricName
 }
 
+func suffix(metricName, suffix string) string {
+	str := metricName
+	if *addSuffix {
```

Comment:
Let's not make this code base even uglier by directly reading flag values from here.

Make it an `Exporter` method instead and pass the desired value into `NewExporter`?